src/main/resources/hudson/plugins/performance/PerformanceReportMap/index.jelly

```diff
@@ -19,9 +19,7 @@
          <j:forEach var="uriReport" items="${performanceReport.getUriListOrdered()}">
            <tr class="${h.ifThenElse(uriReport.failed,'red','')}">
              <td class="left">
-               <a href="./uriReport/${uriReport.encodeUriReport()}">
-               <st:out value="${uriReport.getShortUri()}" />
-               </a>
+               <st:out value="${uriReport.getStaplerUri()}" />
```

Comment:
Well, I admit it was probably not the best approach. The function that generated this wrapper link was removed in one of the previous commits marked as performance optimisation. Quite a lot of code was reorganised during that performance optimisations so I am not totally sure if this functionality was still meant to be kept. My impression was that removing it was a part of optimisation on its own and that this wrapper was just forgotten.